pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## 🎫 Ticket
+
+Link to the relevant jira ticket.
```

Comment:
```suggestion
Link to the relevant Jira ticket.
```
It's a proper noun?

Reply:
or drop Jira entirely, and just say "ticket"?
```suggestion
Link to the relevant ticket.
```

Reply:
I'll drop it entirely.